go.mod

```diff
@@ -109,6 +109,10 @@ require (
 	sigs.k8s.io/kustomize/kustomize/v5 v5.3.0
 )
 
+require github.com/aws/aws-sdk-go-v2/service/secretsmanager v1.25.5
```

Comment:
Please put this new dependenencies into existing blocks in go.mod, don't introduce a new ones.

Reply:
Alright.